README.md

```diff
@@ -159,7 +147,7 @@ Guest Operating System:
 
 ### Public
 * [containers@vmware.com](cna-storage <containers@vmware.com>)
-* [Issues] (https://github.com/vmware/docker-volume-vsphere/issues)
+* [Issues](https://github.com/vmware/docker-volume-vsphere/issues)
```

Comment:
While you are there, can you update " # Known Issues". All of them are already fixed. May be add following-
- Multi-tenancy feature is limited to single ESX #1032